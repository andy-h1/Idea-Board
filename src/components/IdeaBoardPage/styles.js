import styled from 'styled-components';

export const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  place-items: center;
  font-family: 'Noto Sans JP', sans-serif;
  border: 5px solid #f7dc6f;
  background-color: #f4d03f;
  margin: 1em;
`;

export const Title = styled.h1`
  text-align: center;
  padding: 1em;
`;
