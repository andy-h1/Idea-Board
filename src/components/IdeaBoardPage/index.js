import React, { useContext } from 'react';
import { IdeaContext } from '../../contexts/IdeaContext';
import * as S from './styles';

export const IdeaBoardPage = () => {
  const { ideas } = useContext(IdeaContext);

  return (
    <>
      <S.Wrapper>
        <S.Title>Idea Board Page</S.Title>
        <p> Currently you&apos;ve come up with {ideas.length} ideas!!</p>
      </S.Wrapper>

    </>
  );
};
