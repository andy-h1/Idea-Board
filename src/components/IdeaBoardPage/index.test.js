import React from 'react';
import { render } from '@testing-library/react';
import IdeaBoardPage from '.';

describe('Ideas Page Component', () => {
  it('should render', () => {
    const { container } = render(<IdeaBoardPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
