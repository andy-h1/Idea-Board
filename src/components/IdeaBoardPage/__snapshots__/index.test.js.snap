// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ideas Page Component should render 1`] = `
<div
  class="sc-bdnylx wNCQn"
>
  <h1
    class="sc-gtssRu cKXJs"
  >
    Ideas Board
  </h1>
  <p>
    Currently you've come up with 
    0
     ideas!!
  </p>
</div>
`;
